lutris/sysoptions.py

```diff
@@ -161,6 +161,34 @@ def get_optirun_choices():
         'advanced': True,
         'help': "Environment variables loaded at run time"
     },
+    {
+        'option': 'play_script',
+        'type': 'file',
+        'label': 'Play Script',
+        'advanced': True,
+        'help': ("Choose a script to execute with script icon")
```

Comment:
WAT

Reply:
I taught it would make sense if u saw lutris xD
![script](https://user-images.githubusercontent.com/22427111/47703548-ec84f200-dc20-11e8-80da-371e2fb3d1b1.png)
Obviously the wording is bad though, i'll try to figure out something else

Reply:
I'm not adding that button.